Test/README.md

```diff
@@ -126,6 +126,15 @@ Run selected test:
 Invoke-Pester -Script @{ Path = ".\TunnellingWithAgent.Tests.ps1"; Parameters = @{ TestenvConfFile = "testenv-conf.yaml"}; } -TestName 'Tunnelling with Agent tests'
 ```
 
+### Running tests on an already deployed environment
+
+Smoke tests can be run on an already deployed environment (for example by contrail-ansible-deployer). Setting PrepareEnv
+flag to false will cause tests to not setup any services and become independant from deployment method:
+
+```
+ Invoke-Pester -Tag 'Smoke' -Script @{ Path = './Tests/Network/*'; Parameters = @{ PrepareEnv = $false; TestenvConfFile="testenv-conf.yaml" };}
```

Comment:
That would mean that apart from Tags we would also have to pass PrepareEnv flag to Invoke-ProductTests. Don't you think that would clutter it?

Reply:
Or, we can assume that -Smoke flag will append PrepareEnv to Invoke-PesterTests. Or we could not assume that. There are many options here.

How about `Invoke-ProductTests -UsingExistingDeployment`, which would set PrepareEnv and Smoke?

My point is that running smoke tests on existing env seems like a good thing to have a script for, because we want devs to use it easily